srcpkgs/pinephone-base/template

```diff
@@ -1,15 +1,22 @@
 # Template file for 'pinephone-base'
 pkgname=pinephone-base
-version=0.1
+version=0.2
 revision=1
 archs="aarch64*"
 build_style=meta
-depends="pinephone-kernel pinephone-uboot"
+depends="pinephone-kernel pinephone-uboot alsa-ucm-conf"
 short_desc="Void Linux PinePhone platform package"
 maintainer="John Sullivan <jsullivan@csumb.edu>"
-license="Public Domain"
+license="MIT"
```

Comment:
Hmm why did this change?  Usually the base-files metapackage should be public-domain.  If there are other files that need to be included under other licenses, you can make another package for them.

Reply:
Yeah, there's some scripts/configs I borrowed from postmarketos. I'll put them in a separate package then, maybe `pinephone-utils`?

Reply:
Yeah that sounds like a good name.

Reply:
Out of curiosity, could we have pmos as an upstream instead of vendoring the scripts in? (assuming you aren't changing them much).

Reply:
That might be possible, the scripts are sourced from their source packages repo, so I'd need to split them out somehow. However, I think any changes I made can be re-done in a ~10 line patch.